packages/choreography-lint/tests/no-renaming-operator.test.js

```diff
@@ -0,0 +1,33 @@
+const {RuleTester} = require("eslint");
```

Comment:
I actually prefer storing tests in the same directory as the source code, but it may be just me